ui/app/routes/jobs/job/definition.js

```diff
@@ -1,14 +1,24 @@
 import Route from '@ember/routing/route';
 
 export default class DefinitionRoute extends Route {
-  model() {
+  async model() {
     const job = this.modelFor('jobs.job');
     if (!job) return;
 
-    return job.fetchRawDefinition().then((definition) => ({
+    const definition = await job.fetchRawDefinition();
+
+    const definitionWithoutSpecification = { ...definition };
+    delete definitionWithoutSpecification.Specification;
+
+    const specification = await new Blob([
+      definition?.Specification?.Definition,
+    ]).text();
```

Comment:
Can you help me understand what this is accomplishing? `delete prop` outside of a serializer has me wondering if this is the optimal place for this.

Reply:
`fetchRawDefinition` specifically is made to skip the Ember Data and thus should not be serialized because we serialize payloads to format them before they enter the Ember Data store.

That leaves me with 2 places to handle this concerns:  the Route who makes the `fetch` request outside of `Ember Data` or creating a `Service` and extract our logic there. I opted for minimally changing the API surface and handling the concern where we are loading the data.

Reply:
Okay, so why are we deleting `.Specification` here?

What's the format of `definition` ? Why does `Specification` need to be removed from it? I'm struggling to understand what this model is trying to accomplish.

Reply:
We're doing what we would do in our `Serializer` here. We don't require this field. We could keep it, as we're not concerned with it and it won't break anything.